.gitignore

```diff
@@ -37,3 +37,6 @@
 .DS_Store
 /.idea/*.xml
 /.idea/KlinicCon-Backend.iml
+
+# Ignore credential for decrypting credentials and more.
+/config/credentials.yml.enc
```

Comment:
Undo changes from this file. It is not part of this PR.

Reply:
done.